scripts/mirror.js

```diff
@@ -223,6 +223,10 @@ const bumpChromeAndroid = (originalData, sourceData, source) => {
  * @returns {SupportStatement}
  */
 const bumpEdge = (originalData, chromeData, ieData) => {
+  if (Array.isArray(originalData)) {
+    return originalData.map(d => bumpEdge(d));
```

Comment:
What about the `chromeData` and `ieData` arguments in this case? If they're arrays, should the data be used? How about if they're not arrays? Just ignoring the data seems odd, although I don't remember now how any of this works :)